src/components/Select/Select.tsx

```diff
@@ -766,6 +767,9 @@ export const Select: FC<SelectProps> = React.forwardRef(
       inputWidth: inputWidth,
       iconButtonProps: !readonly
         ? {
+            ariaLabel: toggleButtonAriaLabel
+              ? toggleButtonAriaLabel
+              : 'Toggle dropdown',
```

Comment:
this is the part where you were apprehensive about getting locale involved?

Reply:
Yes, at least for now. Will add the Locale for Select after v2.49.2